resources.html.slim

```diff
@@ -121,7 +121,7 @@ div(ng-app="search")
                 option(value="false") Relevance
                 option(value="true") Most Recent
               | and show
-              select(ng-model="params.size" ng-change="updateSearch(); resetPagination();")
+              select(class="results-count" ng-model="params.size" ng-change="updateSearch(); resetPagination();")
```

Comment:
@adelasofia could you cast your eyes over this layouts page as I added some classes to this page in order to test it. If its OK that will give someone the go-ahead to merge. Thanks